app/views/attachments/index.html.erb

```diff
@@ -29,3 +29,18 @@
     </div>
   </div>
 </div>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% attachments = @edition.file_attachment_revisions %>
+
+    <% if attachments.any? %>
```

Comment:
Missing indentation for line 37 under the if block

Reply:
Thanks sorted